### PR TITLE
Refactor mutations.js of cell module

### PR DESCRIPTION
**Pull request by MaikaRenee**

### CODE REVIEW

File: src/store/modules/cells/mutations.js [hunk 1]
```diff
@@ -32,27 +79,30 @@ const mutations = {
...
+    // Update states
     state.cells = boardClone
     state.firstClick = null
   },
 
   mHighlight: (state, coords) => {
     // If there is already a previously highlighted square, cancel it 
     if (state.firstClick !== null) {
-      // console.log("calling munhighlight")
       mutations.mUnhighlight(state, coords)
```
Comment (byronnill) [inline src/store/modules/cells/mutations.js]: remove coords here also right

Comment (byronnill) [reply src/store/modules/cells/mutations.js]: same for in handleIllegal..

Comment (MaikaRenee) [reply src/store/modules/cells/mutations.js]: Why? Not sure I'm following

Comment (MaikaRenee) [reply src/store/modules/cells/mutations.js]: Made a new push. Please check

**Pull request merged**